src/test/java/net/minecraftforge/test/PermissionTest.java

```diff
@@ -0,0 +1,156 @@
+package net.minecraftforge.test;
+
+import net.minecraft.block.Block;
+import net.minecraft.block.state.IBlockState;
+import net.minecraft.command.CommandBase;
+import net.minecraft.command.CommandException;
+import net.minecraft.command.ICommandSender;
+import net.minecraft.command.WrongUsageException;
+import net.minecraft.entity.player.EntityPlayerMP;
+import net.minecraft.nbt.NBTTagCompound;
+import net.minecraft.server.MinecraftServer;
+import net.minecraft.tileentity.TileEntity;
+import net.minecraft.util.math.BlockPos;
+import net.minecraft.util.math.ChunkPos;
+import net.minecraft.util.text.TextComponentString;
+import net.minecraftforge.fml.common.Mod;
+import net.minecraftforge.fml.common.event.FMLServerStartingEvent;
+import net.minecraftforge.server.permission.PermissionAPI;
+import net.minecraftforge.server.permission.context.ContextKey;
+import net.minecraftforge.server.permission.context.PlayerBlockContext;
+import net.minecraftforge.server.permission.context.PlayerChunkContext;
+
+@Mod(modid = PermissionTest.MOD_ID, name = "PermissionTest", version = "1.0.0")
+public class PermissionTest
+{
+    public static final String MOD_ID = "PermissionTest";
+
+    //'Bob' Claim Chunk 0,0
+    //'Sally' Trys place block at 0,0,0 while standing at 100,0,100
+
+    @Mod.Instance(PermissionTest.MOD_ID)
+    public static PermissionTest inst;
+
+    @Mod.EventHandler
+    public void onServerStarted(FMLServerStartingEvent event)
+    {
+        event.registerServerCommand(new CommandPermissionTest());
+    }
+
+    public static class Permissions
+    {
+        public static final String CLAIM_CHUNK = "chunk.claim";
+        public static final String UNCLAIM_CHUNK = "chunk.unclaim";
+        public static final String SET_BLOCK = "block.set";
+        public static final String READ_TILE = "tileentity.read";
```

Comment:
Also, we talked about this on irc, we want to have a registry of possible config values with default values if possible so that we can generate atleast a template config.